12_Highest_Divisible_Triangle.py

```diff
@@ -0,0 +1,48 @@
+import math
+#not using this one
+def get_triangle_number(nth_term):
+    i = nth_term
+    current_sum = 0
+    while(i > 0):
```

Comment:
apparently while loops are slow in python. It's preferable to `for` over a `range` if you know up front what the range is, like here.

Reply:
oh yea ok, you talked about this on slack.